lib/engine/game/g_1849/step/buy_e_token.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G1849
+      module Step
+        class BuyEToken < Engine::Step::Base
+          def actions(entity)
+            actions = []
+            actions += %w[choose pass] if can_buy_e_token?(entity) && entity == current_entity
```

Comment:
The step shouldn't use auto actions and conditional actions here. Since `can_buy_e_token?` is relatively cheap, remove the auto action logic.

Reply:
Done

Reply:
done